Documentation/teaching/labs/memory_mapping.rst

```diff
@@ -446,7 +446,7 @@ that called the :c:func:`read` on that file.
 
 .. hint:: Use :c:func:`proc_create`. For the mode parameter, use 0,
           and for the parent parameter use NULL. Use
-          :c:func:`my_proc_file_ops` for operations.
+          :c:data:`struct file_operations my_proc_file_ops` for operations.
```

Comment:
Hmm, this does not seem like it is rendering properly:
![image](https://user-images.githubusercontent.com/213300/97111022-23ddea80-16e5-11eb-9436-3ce7af0e109e.png)

I think the correct fix is ":c:data:`my_proc_file_ops`".  In this case we can even drop the :c:func: part since we are only use this to create cross-references to kernel API documentation which is not the case here.